scripts/common_tests.py

```diff
@@ -96,7 +98,7 @@ def test_mac_style(self):
         """
         for font in self.fonts:
             font_name = font_data.font_name(font)
-            bold = ('Bold' in font_name) or ('Black' in font_name)
+            bold = 'Bold' in font_name
```

Comment:
Should the bold bit be set for Black? Is there a technical reason why this is important? I had come to the conclusion that while Bold is a variation of Regular (they share the same family name "Roboto"), Black is in some sense it's own style (it can't share the same family name and is instead "Roboto Black").

This is consistent with how we actually output the fonts -- if we want Black to be considered a bold font then we need to change some logic in our scripts.